### PR TITLE
[Frontend] Bugfixes in submission meta attributes css

### PR DESCRIPTION
**Pull request by Ram81**
### Description

This PR fixes css inconsistency in submission meta attributes fields. 
Screen Shot 2020-10-16 at 3 00 14 PM

### CODE REVIEW

**Review by RishabhJain2018 — CHANGES_REQUESTED** [review]
Can we please make the new attributes to look similar to that of previous ones, i.e. label has a smaller font size and lighter font color.

**Comment by Ram81** [comment]
Screen Shot 2020-10-17 at 11 57 07 AM

File: frontend/src/views/web/challenge/submission.html [hunk 1]
```diff
@@ -291,62 +291,64 @@ <h5 class="w-300">Make Submission</h5>
...
-                                                                <span class="fs-16">{{option}}</span>
-                                                            </md-radio-button>
-                                                        </md-radio-group>
+                                                    <div class="align-left col s10">
+                                                        <label class="label-radio" for="{{attribute.name}}">{{attribute.name}} ({{attribute.description}}) <span ng-if="attribute.required === true" class="color-orange">*</span></span></label>
+                                                        <div class="input-field">
+                                                            <md-radio-group ng-model="attribute.value" id="{{attribute.name}}" name="{{attribute.name}}">
+                                                                <md-radio-button ng-repeat="option in attribute.options" ng-value="option" ng-model="attribute.value">
+                                                                    <span class="fs-16">{{option}}</span>
```
Comment (RishabhJain2018) [inline frontend/src/views/web/challenge/submission.html:318]: The font-size for the other options (answers not labels) is `0.9em` IIRC. Can you please make sure that this is same as that.

**Review by RishabhJain2018 — CHANGES_REQUESTED** [review]

File: frontend/src/views/web/challenge/submission.html (hunk 1 above)
Comment (Ram81) [reply frontend/src/views/web/challenge/submission.html:318]: It `1.0em` for other option element from create challenge page. Which is same as these options

Comment (RishabhJain2018) [reply frontend/src/views/web/challenge/submission.html:318]: What about the other submission meta attributes, what is the value for them?

Comment (Ram81) [reply frontend/src/views/web/challenge/submission.html:318]: It is the same

Comment (RishabhJain2018) [reply frontend/src/views/web/challenge/submission.html:318]: perfect! Thanks for checking.

**Review by RishabhJain2018 — APPROVED** [review]

**Pull request merged**